source/beamline/AMPVControl.cpp

```diff
@@ -50,6 +56,17 @@ AMReadOnlyPVControl::AMReadOnlyPVControl(const QString& name, const QString& rea
 
 }
 
+
+void AMReadOnlyPVControl::enableLimitMonitoring()
```

Comment:
Just curious, is there ever a case when we wouldn't want PV controls to monitor limits?